openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -346,28 +347,31 @@ def check_group_D(
     def check_group_E(
         self, permstore: PermissionVarStore, fields: List[str], instance: Dict[str, Any]
     ) -> None:
-        """Check Group E organization_management_level: OML level necessary.
-        Also checks if user has higher oml."""
+        """Check Group E organization_management_level: OML level necessary"""
         if fields and permstore.user_oml < OrganizationManagementLevel.CAN_MANAGE_USERS:
             raise MissingPermission(OrganizationManagementLevel.CAN_MANAGE_USERS)
-        user = self.datastore.get(
-            FullQualifiedId(Collection("user"), instance["id"]),
-            ["organization_management_level"],
-        )
-        if (
-            fields
-            and OrganizationManagementLevel(user.get("organization_management_level"))
-            > permstore.user_oml
-        ):
-            raise PermissionDenied(
-                f"Your organization management level is not high enough to reset a Level of {user.get('organization_management_level')}!"
-            )
 
     def check_group_F(self, fields: List[str]) -> None:
         """Group F: OML SUPERADMIN necessary, which is checked before"""
         if fields:
             raise MissingPermission(OrganizationManagementLevel.SUPERADMIN)
 
+    def _check_for_higher_OML(
+        self, permstore: PermissionVarStore, instance: Dict[str, Any]
+    ) -> None:
+        if "id" in instance:
+            user = self.datastore.get(
+                FullQualifiedId(Collection("user"), instance["id"]),
+                ["organization_management_level"],
+            )
+            if (
+                OrganizationManagementLevel(user.get("organization_management_level"))
+                > permstore.user_oml
+            ):
+                raise PermissionDenied(
+                    f"Your organization management level is not high enough to reset a Level of {user.get('organization_management_level')}!"
```

Comment:
error message does not match the error

Reply:
Update to f`"Your organization management level is not high enough to change a user with a Level of {user.get('organization_management_level')}!"`